tests/unittest_client.py

```diff
@@ -85,22 +85,22 @@ def test_7(self):
         with self.assertRaises(OperationNotAllowed):
             self.write_c.get_label_row(LABEL_ID)
 
-    def test_8(self):
-        new_test_blurb = TEST_BLURB.copy()
-        wrong_id = "97681cfa-0c3a-4ebe-aba3-f17e136fc231"
-        new_test_blurb["data_units"][DATA_ID] = new_test_blurb["data_units"][wrong_id]
-        del new_test_blurb["data_units"][wrong_id]
-        new_test_blurb["data_units"][DATA_ID]["data_hash"] = DATA_ID
-        blurb = self.rw_c.save_label_row(LABEL_ID, TEST_BLURB)
-        assert blurb is True
-
-    def test_10(self):
-        with self.assertRaises(AuthorisationError):
-            self.rw_c.save_label_row("test", TEST_BLURB)
-
-    def test_11(self):
-        with self.assertRaises(OperationNotAllowed):
-            self.read_c.save_label_row(LABEL_ID, TEST_BLURB)
+    # def test_8(self):
```

Comment:
We could use [this pytest feature](https://docs.pytest.org/en/latest/how-to/skipping.html#skipping-test-functions) to be more transparent to skipping.

Reply:
Although the ones in unittest don't use pytest

Reply:
done